### DISCUSSION TITLE
MMT Events data

### DISCUSSION THREAD

**Discussion by soile-cyf** [ec2u/data#10] [Q&A] [answered]
Just wondering about the status of the integrations.
- Turku University:  is this integration finalised or is there still something you are working?
- Turku City: How about this one, is the integration finalized or are you still working on it?

Thanks @knoan 

P.S. Sorry if this question is in the wrong place.

FYI @juhovainio @hmaskat17

**Comment by knoan** [comment, marked as answer]
> Just wondering about the status of the integrations.
> 
> * Turku University:  is this integration finalised or is there still something you are working?
> * Turku City: How about this one, is the integration finalized or are you still working on it?

Both integration are completed, with possible annotated minor improvements and pending a general revision of time zone handling in start/end dates (#6); you can get an idea of the material being ingested at:

* https://github.com/ec2u/data/blob/31f1f2f3f563d17e2e4a65f1ea1a48d0f069b5c1/src/main/java/eu/ec2u/data/tasks/events/EventsTurkuUniversity.java#L196
* https://github.com/ec2u/data/blob/31f1f2f3f563d17e2e4a65f1ea1a48d0f069b5c1/src/main/java/eu/ec2u/data/tasks/events/EventsTurkuCity.java#L150

Current integration notes report the following info missing from KH:

* `schema:image` for Turku University
* `schema:location` for Turku City

but both fields are either missing/unpopulated in the primary sources.

> P.S. Sorry if this question is in the wrong place.

Migrated discussion to https://github.com/ec2u/data

**Comment by soile-cyf** [comment]
We had a project group meeting today 18th May. Britta, from Jena university, said that when we went live there were more events integrated to MMT. Also Jena international university events were then visible and currently those are not.
I promised to check with you if some some integrations have been taken away or do you know why there might have been events from multiple Jena calendars and now there are not?
@knoan 

@juhovainio
@hmaskat17
